static/js/utils.js

```diff
@@ -1,5 +1,8 @@
 function alert(type, message, delay) {
   console.info(type, message)
+  if(!message) {
```

Comment:
mhhh… why would you call alert with only 1 argument?
What would be the type?

Reply:
because when an user block the location from his browser, the browser run alert as `alert(message)`
so message is `undefined` and type is a message

Reply:
Are you saying that the browser is calling the `alert()` function by itsself when rejecting geolocation?
It would then make sense, though probably in another PR (as it's not specific to this PR)